File: gems/test-miser/examples/java/src/test/java/dev/testmiser/ClassifierTest.java
package dev.testmiser;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertTrue;

import org.junit.jupiter.api.Test;

class ClassifierTest {
    @Test void positivePrimary() { assertEquals("positive", Classifier.classify(5)); }
    @Test void positiveDuplicate() { assertEquals("positive", Classifier.classify(5)); }
    @Test void high() { assertEquals("high", Classifier.classify(11)); }
    @Test void nonpositive() { assertEquals("nonpositive", Classifier.classify(0)); }
    @Test void smoke() { assertTrue(true); }
}

File: gems/test-miser/examples/kotlin/src/test/kotlin/dev/testmiser/ClassifierTest.kt
package dev.testmiser

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Assertions.assertTrue
import org.junit.jupiter.api.Test

class ClassifierTest {
    @Test fun positivePrimary() = assertEquals("positive", Classifier.classify(5))
    @Test fun positiveDuplicate() = assertEquals("positive", Classifier.classify(5))
    @Test fun high() = assertEquals("high", Classifier.classify(11))
    @Test fun nonpositive() = assertEquals("nonpositive", Classifier.classify(0))
    @Test fun smoke() = assertTrue(true)
}
